test/core/VolOracle.ts

```diff
@@ -20,23 +20,22 @@ describe("VolOracle", () => {
   const ethusdcPool = "0x8ad599c3A0ff1De082011EFDDc58f1908eb6e6D8";
   // const wbtcusdcPool = "0x99ac8cA7087fA4A2A1FB6357269965A2014ABc35";
 
-  const weth = "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2";
-  // const wbtc = "0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599";
-  const usdc = "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48";
-
   before(async function () {
     [signer] = await ethers.getSigners();
     const VolOracle = await getContractFactory("VolOracle", signer);
     const TestVolOracle = await getContractFactory("TestVolOracle", signer);
 
-    oracle = await VolOracle.deploy(ethusdcPool, weth, usdc, PERIOD);
-    mockOracle = await TestVolOracle.deploy(ethusdcPool, weth, usdc, PERIOD);
+    oracle = await VolOracle.deploy(PERIOD);
+    mockOracle = await TestVolOracle.deploy(PERIOD);
     // oracle = await VolOracle.deploy(ethusdcPool, weth, usdc);
   });
 
   describe("twap", () => {
     it("gets the TWAP for a period", async function () {
-      assert.equal((await oracle.twap()).toString(), "2427732358");
+      assert.equal(
+        (await oracle.twap(ethusdcPool)).toString(),
+        "411907019541423"
+      );
```

Comment:
This change is because we're quoting the USDC/ETH price instead of ETH/USDC price